hack/generator/pkg/codegen/pipeline_stage.go

```diff
@@ -7,12 +7,16 @@ package codegen
 
 import (
 	"context"
+
 	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
 )
 
+// Types is the set of all types being generated
+type Types = map[astmodel.TypeName]astmodel.TypeDefinition
```

Comment:
I think this should be just `type Types map[astmodel.TypeName]astmodel.TypeDefinition` - the `=` form is more meant for aliasing types from other packages to support gradual refactoring in a large codebase.

Reply:
I thought that that stopped you being able to use `range` iteration but it looks like it doesn’t. Will change 👍